docs/source/recipes/configure-airflow-as-a-runtime.md

```diff
@@ -80,8 +80,8 @@ To deploy Apache Airflow on a new Kubernetes cluster:
    ```
   
 2. Download, review, and customize the [sample `helm` configuration](https://raw.githubusercontent.com/elyra-ai/elyra/master/etc/kubernetes/airflow/helm/values.yaml) (or customize an existing configuration):
```

Comment:
Lets add a blurb here letting folks know that this sample configuration will use the `KubernetesExecutor` by default.

Reply:
done 👍